gtsfm/common/view_frustum.py

```diff
@@ -57,7 +57,8 @@ def get_frustum_vertices_camfr(
             ]
         )
 
-        ray_dirs = compute_pixel_ray_directions_vectorized(uv, self.fx_, self.img_w_, self.img_h_)
+        ray_dirs = compute_pixel_ray_directions_vectorized(
```

Comment:
i think this change is unrelated -- do you mind checking out the change here/resetting this file to HEAD (carefully :-) though)

Reply:
oops. I think my VScode auto formatter did this. Will reset this back to original.